interface/main/tabs/main.php

```diff
@@ -29,13 +29,44 @@
 /* Include our required headers */
 require_once('../../globals.php');
 require_once $GLOBALS['srcdir'].'/ESign/Api.php';
+require_once $GLOBALS['srcdir']."/../vendor/mobiledetect/mobiledetectlib/Mobile_Detect.php";
+
+    $detect = new Mobile_Detect;
+     $device_type = ($detect->isMobile() ? ($detect->isTablet() ? 'tablet' : 'phone') : 'computer');
+    $script_version = $detect->getScriptVersion();
+    
+if (!empty($_GET['desktop'])) {
+    $desktop = $_GET['desktop'];
+    if ($desktop == 1) {
+        $_SESSION['desktop'] = 1;
+    }
+}
+    /**
+     * May need a global here to disable mobile for those who prefer full desktop for their device
+     *      on a regular login, or have a check box at login to set or unset $_SESSION['desktop']?
+     * Otherwise a user on a mobile phone/tablet is auto-redirected to this limited mobile version of the website.
+     * In the footer of the mobile website is a "Desktop" link.
+     * If “Go to full website” link is clicked from Mobile site, it'll redirect mobile user to main website
+     *      by setting $_SESSION['desktop'] above and by-passing this next piece of code.
```

Comment:
Lets definitely make a global called something liked 'Support Mobile Version' at the bottom of the Appearance tab in globals. Default it to on for now (when we put in the codebase, we will default it to off until it's well worked out, supports rtl, etc.). This then allows you to keep working on things without worrying about breaking things.